care/abdm/utils/api_call.py

```diff
@@ -0,0 +1,771 @@
+import json
+import uuid
+from base64 import b64encode
+from datetime import datetime, timedelta, timezone
+
+import requests
+from Crypto.Cipher import PKCS1_v1_5
+from Crypto.PublicKey import RSA
+from django.conf import settings
+from django.core.cache import cache
+from django.db.models import Q
+
+from care.abdm.models import AbhaNumber
+from care.facility.models.patient_consultation import PatientConsultation
+
+GATEWAY_API_URL = settings.ABDM_URL
+HEALTH_SERVICE_API_URL = settings.HEALTH_SERVICE_API_URL
+ABDM_GATEWAY_URL = GATEWAY_API_URL + "/gateway"
+ABDM_TOKEN_URL = ABDM_GATEWAY_URL + "/v0.5/sessions"
+ABDM_TOKEN_CACHE_KEY = "abdm_token"
+
+# TODO: Exception handling for all api calls, need to gracefully handle known exceptions
+
+
+def encrypt_with_public_key(a_message):
+    rsa_public_key = RSA.importKey(
+        requests.get(
+            HEALTH_SERVICE_API_URL + "/v2/auth/cert", verify=False
```

Comment:
`verify=false` is a security issue
why are we doing this?

Reply:
not sure, @vigneshhari why did we use `verify=False`, was it temporary?

Reply:
It was temporary, revert before we merge